neo/SmartContract/Contract.py

```diff
@@ -91,8 +91,8 @@ def Create(redeemScript, parameterList, publicKeyHash):
     @staticmethod
     def CreateMultiSigRedeemScript(m, publicKeys):
 
-        if m < 2 or m > len(publicKeys) or len(publicKeys) > 1024:
-            raise Exception('Invalid keys')
+        if m < 1 or m > len(publicKeys) or len(publicKeys) > 1024:
+            raise Exception('Argument Exception')
```

Comment:
"Argument exception" is perhaps too generic? Would be great to have a more user-friendly message.

Reply:
Yes correct, it now is generic. Reason for that is because previously it would give a misleading message saying "invalid keys" while tit could be he minimum amount of signatures required was too low (e.g. if m < 2). I guess I would have to split it up then.

How about?
```python
if m < 1:
    raise Exception("Minimum required signature count is 1, specified {}.".format(m))

if m > len(publicKeys):
    raise Exception("Invalid public key count. Minimum required signatures is bigger than supplied public keys count.")

if len(publicKeys) > 1024:
   raise Exception("Supplied public key count ({}) exceeds maximum of 1024.".format(len(publicKeys))

```

Reply:
That makes sense